lib/spandex_phoenix/telemetry.ex

```diff
@@ -0,0 +1,34 @@
+defmodule SpandexPhoenix.Telemetry do
+  @moduledoc """
+  Defines
+  """
+
+  @doc "Attaches `:telemetry` event handlers for Phoenix Telemetry events."
+  def attach(opts \\ []) do
+    handlers = %{
+      [:phoenix, :router_dispatch, :start] => &phoenix_router_dispatch_start/4,
+      [:phoenix, :router_dispatch, :stop] => &phoenix_router_dispatch_stop/4
```

Comment:
Are these the only two telemetry events built in to phoenix 1.5?

Reply:
there's a whole bunch. as i said in the main writeup, i think the entire SpandexPhoenix tracing stack can be moved to just telemetry (no more need for `use SpandexPhoenix`).
but as long as we have `use SpandexPhoenix`, it catches the endpoint start/stop/error_rendered situation.

these router dispatch start/stop events just replace the `Phoenix.Instrumenters` config callbacks that emit deprecation warnings in 1.5.

Reply:
in other words, i made this module by just copying the behavior of of `SpandexPhoenix.Instrumenter`. `phoenix_router_dispatch_start/4` is `phoenix_controller_call(:start, ...)`, and `.._stop/4` is `phoenix_controller_call(:stop, ...)`